lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -394,6 +394,34 @@ def test_error(self):
 
 
 @httpretty.activate
+class ThreadViewSetDeleteTest(DiscussionAPIViewTestMixin, ModuleStoreTestCase):
+    """Tests for ThreadViewSet create"""
+    def setUp(self):
+        super(ThreadViewSetDeleteTest, self).setUp()
+        self.url = reverse("thread-detail", kwargs={"thread_id": "test_thread"})
+        self.thread_id = "test_thread"
+
+    def test_basic(self):
+        self.register_get_user_response(self.user)
+        cs_thread = make_minimal_cs_thread({
+            "id": self.thread_id,
+            "course_id": unicode(self.course.id),
+            "username": self.user.username,
+            "user_id": str(self.user.id),
+        })
+        self.register_get_thread_response(cs_thread)
+        self.register_delete_thread_response(self.thread_id)
+        response = self.client.delete(self.url)
+        self.assertEqual(response.status_code, 204)
+        self.assertIsNone(response.data)
```

Comment:
This assertion could pass if the response body were `null` with the content type `application/json`. Use `self.assertEqual(response.body, "")` instead.